doc/man3/X509_dup.pod

```diff
@@ -252,13 +255,10 @@ X509_ALGOR_new,
 X509_ATTRIBUTE_dup,
 X509_ATTRIBUTE_free,
 X509_ATTRIBUTE_new,
-X509_CERT_AUX_dup,
```

Comment:
I don't see the reason of these removals ?

Reply:
We have removed this and four further such `_dup` occurrences because these functions are not really present/exported, so should not be documented there.